assembl/static2/package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "test": "jest",
     "test:watch": "npm test -- --watch",
-    "build": "node --max-old-space-size=8192 ./node_modules/.bin/webpack --progress --colors",
```

Comment:
@ayyazdaniaryan this would break the gitlab build. I don't know why the memory swap was increased for this build command but running the regular Webpack build seems OK.

cc. @cedrtang

Reply:
It's before we switched to the Terser..on smaller sized EC2 machines it was crashing out of memory so I think it's safe to drop now